rkt/rkt.go

```diff
@@ -102,6 +103,21 @@ func main() {
 	os.Exit(cmd.Run(cmd.Flags.Args()))
 }
 
+func maybeAddNewline(s string) string {
+	if !strings.HasSuffix(s, "\n") {
+		s = s + "\n"
+	}
+	return s
+}
+
+func stderr(format string, a ...interface{}) (int, error) {
+	return fmt.Fprintf(os.Stderr, maybeAddNewline(format), a...)
```

Comment:
I prefer [this form](https://github.com/appc/spec/blob/4c3cbeae47980d073828dd263a8c982667855f06/actool/actool.go#L99) for conciseness but this is basically bikeshedding